app/controllers/support_interface/support_users_controller.rb

```diff
@@ -19,6 +19,18 @@ def create
       end
     end
 
+    def confirm_destroy
+      @support_user = SupportUser.find(params[:id])
+    end
+
+    def destroy
+      if FeatureFlag.active?('delete_support_users')
```

Comment:
Return 404 if feature flag is not active?

Reply:
...or as @tvararu suggested we could just drop the feature flag

Reply:
I've removed the feature flag commit, was useful to learn how to use the `rollout` gem in this app.